src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -267,21 +268,33 @@ protected function render(string $view, array $parameters = [], Response $respon
     }
 
     /**
-     * Renders a view for a form.
+     * Renders a view.
```

Comment:
i'd revert to avoid having two methods with the same description

Reply:
updated